sources/packages/web/src/views/student/financial-aid-application/FullTimeApplication.vue

```diff
@@ -1,39 +1,38 @@
 <template>
-  <student-page-container>
+  <student-page-container :full-width="true">
+    <template #header>
+      <header-navigator
+        title="Applications"
+        subTitle="Financial Aid Application"
+      >
+      </header-navigator>
+    </template>
     <body-header>
       <template #actions>
         <v-row class="m-0 p-0 float-right">
           <v-btn
             color="primary"
-            v-if="!notDraft"
-            v-show="!isFirstPage && !submittingApplication"
+            v-if="!notDraft && !isFirstPage && !submittingApplication"
```

Comment:
Why was it changed?

Reply:
Both v-if and v-show are there to show/hide the button. For me it is hard to understand if you have 2 places to check the logic. Also I prefer using v-if when possible cause it doesn't render the component in case of false. v-show renders it and hides it in case of false, which makes the component to flash in the screen when rendering.

Reply:
I feel we have either one of them to avoid confusion.

Reply:
`v-if`` and `v-show` serves different purposes. I would be ok to not use them but since they were in place it would be a downgrade.
There is a technical reason why both were used but I am not sure if there is a technical reason to remove them.
 Since it is a minor issue I will not ask to change it back.

Reply:
> I feel we have either one of them to avoid confusion.

The confusion can be avoided with a quick check of the documentation.

Reply:
`v-if and v-show` have different purposes. But I don't remember the reason here.

Reply:
Keep on one way or another and let's move on 😉 
There will be no impact and we can shorten the discussion.